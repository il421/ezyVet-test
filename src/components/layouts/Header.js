import React from 'react';
import logo from '../../assets/img/logo.png';
import '../../styles/layouts/header.css';

export const App = () => {
  return (
    <header className="header">
      <img src={ logo } className="header__logo" alt="EzyVet Logo" width="230" height="60" />
    </header>
  );
};

export default App;
