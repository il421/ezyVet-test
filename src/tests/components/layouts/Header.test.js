import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../../components/layouts/Header.js';

test('should render Header without crashing', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
