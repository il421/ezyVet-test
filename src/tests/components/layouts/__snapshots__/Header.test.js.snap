// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header without crashing 1`] = `
<header
  className="header"
>
  <img
    alt="EzyVet Logo"
    className="header__logo"
    height="60"
    src="logo.png"
    width="230"
  />
</header>
`;
